vignettes/socialmixr.Rmd

```diff
@@ -177,26 +180,24 @@ contact_matrix(polymod, countries = "United Kingdom", age.limits = c(0, 1, 5, 15
 
 # Filtering
 
-The `filter` argument to `contact_matrix` can be used to select particular participants or contacts. For example, in the `polymod` dataset, the indicators `cnt_home`, `cnt_work`, `cnt_school`, `cnt_transport`, `cnt_leisure` and `cnt_otherplace` take value 0 or 1 depending on where a contact occurred:
-
-```{r message=FALSE, warning=FALSE}
-head(polymod$contacts)
-```
+The function `contact_matrix()` enables the filtering of social contact data. This can for example be used to select contacts reported at school or to select long-duration contacts. Any filter can be applied to the data, if given as a list of the form (column=filter_value). As such, only contacts that have 'filter_value' in 'column' will be considered for the generated contact matrix.
```

Comment:
I prefer the text as is as it's a bit more explicit - did you mean to change this?

Reply:
I thought it would be useful to add more information on how the filtering is proceeded, though agree that it became less explicit. I hope the latest edits are OK.